src/library-authoring/components/ContainerCard.tsx

```diff
@@ -0,0 +1,92 @@
+import { FormattedMessage, useIntl } from '@edx/frontend-platform/i18n';
+import {
+  ActionRow,
+  Dropdown,
+  Icon,
+  IconButton,
+} from '@openedx/paragon';
+import { MoreVert } from '@openedx/paragon/icons';
+import { Link } from 'react-router-dom';
+
+import { type ContainerHit, PublishStatus } from '../../search-manager';
+import { useComponentPickerContext } from '../common/context/ComponentPickerContext';
+import { useLibraryContext } from '../common/context/LibraryContext';
+import BaseComponentCard from './BaseComponentCard';
+import messages from './messages';
+
+type ContainerMenuProps = {
+  containerHit: ContainerHit,
```

Comment:
I think it should be called just `hit`, following the pattern of other cards like [ComponentCard](https://github.com/openedx/frontend-app-authoring/pull/1742/files#diff-49e613dfd8b3d4c99b6cb27fe426de4977f8701af3c3c262fcf4e72d3d5346a0R183) and [CollectionCard](https://github.com/openedx/frontend-app-authoring/pull/1742/files#diff-e5eaa334df2d12ae99abaec10fa351e88bc07e1151cb72686bd1edd8741c7f74R121)

Reply:
Done! a863cf34e1a4ea365e991df82eddced649cd8e49